components/properties/src/props.rs

```diff
@@ -2455,3 +2502,107 @@ impl_value_getter! {
         pub fn get_enum_to_long_name_mapper() / enum_to_long_name_mapper() -> PropertyEnumToValueNameLinearMapper / PropertyEnumToValueNameLinearMapperBorrowed;
     }
 }
+#[cfg(test)]
+mod test_enumerated_property_completeness {
+    use super::*;
+    use alloc::collections::BTreeMap;
+
+    fn check_enum<'a>(
+        lookup: &PropertyValueNameToEnumMapV1<'static>,
+        consts: impl IntoIterator<Item = &'a (&'static str, u16)>,
+    ) {
+        let mut data: BTreeMap<_, _> = lookup
+            .map
+            .iter_copied_values()
+            .map(|(name, value)| {
+                (
+                    value,
+                    (
+                        String::from_utf8(name.as_byte_slice().to_vec()).unwrap(),
+                        "Data",
+                    ),
+                )
+            })
+            .collect();
+
+        let consts: BTreeMap<_, _> = consts
```

Comment:
Note: I think you don't need to collect the second iterator into a `BTreeMap` because you basically only use it as an iterator down below. Not a problem here since this is test code, but in library code I would be more strict.

Reply:
Fixed :)